vendor/src/github.com/docker/notary/client/delegations.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"encoding/json"
```

Comment:
Yep, we tried switching everything to Canonical JSON and realized we weren't perfect about it upfront. We have some work to do there but Go's serialization is consistent and reliable so it's not critical because notary is an opinionated version (only in minor but incompatible implementation details) of TUF that isn't expected to work with other clients/servers.